scrapers/scrape_matrix.py

```diff
@@ -23,7 +23,7 @@
     'GR': ['Confirmed cases', 'Deaths', 'Hospitalized'],
     'JU': ['Confirmed cases', 'Deaths', 'Hospitalized', 'ICU'],
     'LU': ['Confirmed cases', 'Deaths', 'Hospitalized', 'ICU'],
-    'NE': ['Hospitalized', 'Vent'],
+    'NE': ['Confirmed cases', 'Deaths', 'Hospitalized', 'ICU', 'Vent'],
```

Comment:
This will fail most of the time, since NE does not publish this data every day. So if you define it as an "expectation" here, the scraper will fail and update no data.

Reply:
Reverted